src/constants.ts

```diff
@@ -50,6 +50,7 @@ export const TAXONOMY_CUSTOM_FIELD_UNGROUPED_RESOURCE = 'ungrouped';
 export const TAXONOMY_CUSTOM_FIELD_SUBJECT_FOR_CONCEPT = 'forklaringsfag';
 export const TAXONOMY_CUSTOM_FIELD_SUBJECT_OLD_SUBJECT_ID = 'old-subject-id';
 export const TAXONOMY_CUSTOM_FIELD_SUBJECT_CATEGORY = 'subjectCategory';
+export const TAXONOMY_CUSTOM_FIELD_SUBJECT_YEAR = 'subjectYear';
```

Comment:
Ellers i løsninga er grades brukt for dette. Kanskje være konsekvent og kalle det for det.